go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
 	golang.org/x/tools v0.1.5 // indirect
+	google.golang.org/protobuf v1.27.1
```

Comment:
nit, looks like `go mod tidy` is needed (`go test` complained when I ran it locally)

Reply:
done